diff --git a/Examples/JavascriptPlayground.uproject b/Examples/JavascriptPlayground.uproject
index 42eeaa6..2dbbc71 100644
--- a/Examples/JavascriptPlayground.uproject
+++ b/Examples/JavascriptPlayground.uproject
@@ -1,6 +1,14 @@
 {
 	"FileVersion": 3,
-	"EngineAssociation": "4.12",
+	"EngineAssociation": "{1C7B2890-4893-6EAA-0BC3-588EAED42C4F}",
 	"Category": "",
-	"Description": ""
+	"Description": "",
+	"Modules": [
+		{
+			"Name": "JavascriptPlayground",
+			"Type": "Runtime",
+			"LoadingPhase": "Default"
+		}
+	]
+
 }
\ No newline at end of file
