From 773c200b697a9d6cddacb1da02c5663a3b1e84b1 Mon Sep 17 00:00:00 2001
From: Hyeoncheol Cho <crocuis@gmail.com>
Date: Sun, 22 Nov 2015 05:39:03 +0900
Subject: [PATCH] add gitignore

---
 Examples/.gitignore | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 Examples/.gitignore

diff --git a/Examples/.gitignore b/Examples/.gitignore
new file mode 100644
index 0000000..576321f
--- /dev/null
+++ b/Examples/.gitignore
@@ -0,0 +1,3 @@
+Intermediate
+Saved
+DerivedDataCache
